src/html/partials/concept.html

```diff
@@ -1,4 +1,73 @@
 <section class="concept-section">
-  <h2 class="sm-title">The Concept</h2>
-  <p class="bg-intro">Meet Florence</p>
+
+  <div class="concept-section__intro">
+    <h2 class="sm-title">The Concept</h2>
+    <p class="bg-intro">Meet Florence</p>
+
+    <p>Our explorations led to the concept of Florence — a personal companion that would help patients understand and manage their medication regimen. A chatbot at heart, patients would converse with her through text and picture messages.</p>
+      
+    <p>If developed, a simple, familiar interface would hide her powerful intelligence. Behind the scenes, machine learning and behavioral modeling would enable her to offer a tailored, human experience based on each patient’s needs.</p>
+  </div>
+
+  <ul class="concept-section__list">
+    <li class="concept-section__item">
+      <div class="concept-section__item__card">
+        <h3>She Gets To Know You.</h3>
+        <p>Florence strikes up a conversation, learning details about your life, health, goals, and preferences in an engaging, human way.</p>
+        <p>This conversation unfolds over time, making this feel less like twenty questions and more like the start of a relationship.</p>
+      </div>
+      <img src="../../images/sms/Text_SMS_1.png" alt="Hi! I'm Florence. What's your name?">
+    </li>
+
+    <li class="concept-section__item">
+      <div class="concept-section__item__card">
+        <h3>She Keeps Track of Your Regimen.</h3>
+        <p>Florence helps you remember what to take and when. Image recognition makes adding your medication as easy as taking a picture.</p>
+      </div>
+      <img src="../../images/sms/Text_SMS_2.png" alt="Let's get started adding your medications.">
+    </li>
+    
+    <li class="concept-section__item">
+      <div class="concept-section__item__card">
+        <h3>She Checks Up On You.</h3>
+        <p>As you start taking new medications, Florence will follow up to see if you’re experiencing side effects.</p>
+        <p>She can offer additional information to help you understand if your experience is typical—or if you should talk with your doctor.</p>
+      </div>
+      <img src="../../images/sms/Text_SMS_3.png" alt="It'sbeen three days since you started taking Lisinopril. How're you feeling?">
+    </li>
+    
+    <li class="concept-section__item">
+      <div class="concept-section__item__card">
+        <h3>She Helps You Remember.</h3>
+        <p>With complex regimens, days often blur together. Florence will show you what you’ve taken today—and what’s left to take.</p>
```

Comment:
Should the em-dash between `today` and `and` have spaces around it?

Reply:
[Not in most cases, but newspapers sometimes add spaces.](http://www.thepunctuationguide.com/em-dash.html)